akka-http-core/src/main/resources/reference.conf

```diff
@@ -447,6 +447,11 @@ akka.http {
       # Fail the connection if a sent ping is not acknowledged within this timeout.
       # When zero the ping-interval is used, if set the value must be evenly divisible by less than or equal to the ping-interval.
       ping-timeout = 0s
+
+      # The maximum number of times a connections is attempted
+      # before giving up and returning an error.
+      # Set to zero to retry indefinitely.
+      max-persistent-attempts = 0
```

Comment:
I was surprised by the default to have infinite retries. I'm not sure if it's good or bad...

Reply:
I guess once we have backoff, we'd rather keep it trying indefinitely and just set a maximum backoff interval, so it tries only now-and-then.